R/utils.R

```diff
@@ -297,3 +297,9 @@ format_difftime <- function(time1, time2) {
     sprintf("%.2f %s", z, units)
   }
 }
+
+format_difftime_ms <- function(time1, time2) {
+  z <- as.numeric(time1 - time2, "secs")
+  z <- z * 1000
+  sprintf("%.0f %s", z, "ms")
```

Comment:
Do you really want this as the string "0.1234 ms" and not the number `0.1234`? how does downstream need it?

Reply:
Nope :facepalm: just because I am rushing